eleventy-config.js

```diff
@@ -34,13 +35,26 @@ function notIndex(thing) {
   return !thing.inputPath.includes('index.njk');
 }
 
+function currentEpisodes(episode) {
+  let episode_date = episode && episode.data && episode.data.stuff && episode.data.stuff.date
+
+  // If the episode defines a date and it's after current hide it
+  if (episode_date && episode_date > new Date()) {
+    return false;
+  }
+
+  return true;
```

Comment:
Should probably hide it if it's missing a date on anything else, too.

Reply:
I actually wanted to "fail open" in that case.. only if you specify a date does it check to see if that date is after now

Reply:
I guess that could work, though I'm not sure what a podcast client would do with that.